src/test/data/JsonAddressBookStorageTest/invalidAndValidParticipantAddressBook.json

```
@@ -0,0 +1,43 @@
+{
+  "persons": [
+    {
+      "name": "Valid Person",
+      "phone": "9482424",
+      "email": "hans@example.com",
+      "address": "4th street",
+      "birthDate": "2000-08-02",
+      "notes": [
+        {
+          "content": "Vegetarian",
+          "importance": "VERY_HIGH"
+        }
+      ],
+      "nextOfKins": [
+        {
+          "name": "Janette Yeoh",
+          "phone": "93822312",
+          "tag": "Spouse"
+        }
+      ]
+    },
+    {
+      "name": "Person With Invalid Phone Field",
+      "phone": "948asdf2424",
+      "email": "hans@example.com",
+      "address": "4th street",
+      "birthDate": "2000-08-02",
+      "notes": [
+        {
+          "content": "Vegetarian",
+          "importance": "VERY_HIGH"
+        }
+      ],
+      "nextOfKins": [
+        {
+          "name": "Janette Yeoh",
+          "phone": "93822312",
+          "tag": "Spouse"
+        }
+      ]
+    },
+    }
```

```suggestion
    }
  ]
}
```
Missing closing square bracket